core/src/components/picker-column-internal/test/basic/e2e.ts

```diff
@@ -51,6 +51,28 @@ describe('picker-column-internal', () => {
       expect(activeColumn.innerText).toEqual('23');
     });
 
+    it('should not emit ionChange when the value is modified externally', async () => {
+      const pickerColumn = await page.find('#default');
+      const ionChangeSpy = await pickerColumn.spyOnEvent('ionChange');
+
+      await page.$eval('#default', (el: any) => {
+        el.value = '12';
+      });
+
+      expect(ionChangeSpy).not.toHaveReceivedEvent();
+    });
+
+    it('should emit ionChange when the picker is scrolled', async () => {
+      const pickerColumn = await page.find('#default');
+      const ionChangeSpy = await pickerColumn.spyOnEvent('ionChange');
+
+      await page.$eval('#default', (el: any) => {
+        el.scrollTo(0, 300);
+      });
+
+      expect(ionChangeSpy).not.toHaveReceivedEvent();
```

Comment:
```suggestion
      expect(ionChangeSpy).toHaveReceivedEvent();
```

Reply:
Great catch, I also needed to wait for the event:
```ts
await ionChangeSpy.next();

expect(ionChangeSpy).toHaveReceivedEvent();
```

Confirmed this test passed with the changes and failed against main. I'll wait for CI before finally merging.